### DISCUSSION TITLE
Questions About FS-iA6B Receiver

### DISCUSSION THREAD

**Discussion by mblabla** [OpenI6X/opentx#367] [General]
Hi  , i recently just flashed my Flysky with Openi6x and i have the receiver that came with it , and in the mind time i receive the ExpressLRS module that i need i would like to know if it could be ssuported. 
The actual information in the website says  that their protocol is "AFHDS2A" but in some other places it says "AFHDS 2A" 
and i was basically wondering if any of you have tried before to bind the radio having OpenI6X on it  and were sucessfull to see if i am doing anything wrong during the binding process.

1.  i went to the model menu 
2. i enter in to the setup menu 
3. i hit the Bnd option 
4. Turn on the Receiver with the binding key pressed
But the lights keep flashing as  the bindiong process was not done sucessfully  ("The entired time the Radio was beeping for some reason"); 

Any Suggestions?

**Comment by mblabla** [comment]
@ajjjjjjjj
@marianomd 
 Any suggestion?

**Comment by didinski** [comment]
Hi Mel,

i'm using a ia10b rx and tested a ia6b too. Try a complete reset at the calibration menu.

- turn on the RX with the jumper bind cable
- go to the openi6x bind (model) menu and hit bind
- remove the jumper.
 
works for me every time.

**Reply by mblabla** [reply]
How could i even forget the bind cable on the Jumper , i really don't know how i could even forget that  , i will try it within a couple of minutes  again but  i actually forgot to do that , thanks a lot i will try it then i will let you know my results @didinski

**Comment by mblabla** [comment]
![Binded](https://github.com/OpenI6X/opentx/assets/87794877/97673dbf-47e4-4a05-b1b7-c7ebf2848d05)
Thanks a lot for your help @didinski it worked!!!  i forgot that i had to connect that binding jumper i thought with the button that it has being press it would work.
